tome/achievements/tests/test_post_update_scoresheet.py

```
@@ -0,0 +1,343 @@
+            Achievements(
+                id=AID9,
+                name="Zero or less life",
+                slug="zero-or-less-life",
+                point_value=3,
+            ),
+        ]
+    )
+
+
+@pytest.fixture(scope="function")
+def build_inserted_state(client, build_state) -> dict:
```

Ruff (0.14.11)

Unused function argument: `client`

(ARG001)